cmd/replay.go

```diff
@@ -267,3 +270,90 @@ func runReplayRequest(l logger, projectName, namespace, jobName, startDate, endD
 	}
 	return replayResponse.Id, nil
 }
+
+func replayStatusSubCommand(l logger, conf config.Provider) *cli.Command {
+	var (
+		replayProject string
+		namespace     string
+	)
+
+	reCmd := &cli.Command{
+		Use:     "status",
+		Short:   "get status of a replay using its ID",
+		Example: "optimus replay status replay-id",
+		Long: `
+This operation takes one argument, replay ID[required]
+that generated when starting a replay.
+		`,
+		Args: func(cmd *cli.Command, args []string) error {
+			if len(args) < 2 {
+				return errors.New("job name and replay ID are required")
```

Comment:
Btw, isn't replay id unique, why do we need a job name?

Reply:
yes, it is unique. have added it for the api call: https://github.com/odpf/proton/blob/4bcadc9389fa7e895509fa85cd7a2c25af5f3dc6/odpf/optimus/runtime_service.proto#L192
let me know your thoughts and if you think removing it is more proper

Reply:
For rest call to make it follow the convention I guess we need it there but we are using GRPC here, do we really need it?

Reply:
agreed. updated to only require replay ID here